src/js/components/Video/stories/typescript/TheaterMode.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+
+import { Box, Grommet, Video, grommet } from 'grommet';
+import { Monitor } from 'grommet-icons/icons/Monitor';
+
+const TheaterMode = (props) => {
+  const [theaterMode, setTheaterMode] = useState(false);
+
+  return (
+    <Grommet theme={grommet}>
```

Comment:
Need to remove `<Grommet />`, like in other stories. This might help fix the fact that currently the video is showing full height but small width, which looks weird.

![Screen Shot 2021-12-15 at 9 26 52 AM](https://user-images.githubusercontent.com/11637956/146235184-c818b046-a30e-4750-be7b-518175049fe8.png)